packages/manager/modules/core/src/redirection/redirection.service.js

```diff
@@ -35,7 +34,7 @@ export default class RedirectionService {
   }
 
   static validate(url) {
-    return some(Object.values(Environment.getApplicationURLS()), (appURL) =>
+    return some(Object.values(this.coreConfig.getApplicationURLS()), (appURL) =>
```

Comment:
Same here, the method becomes not static

Reply:
Fixed, thanks.